### PR TITLE
[W5.11][W13-A1] Karen Frilya Celine 

### PR DESCRIPTION
**Pull request by karen-celine**
Add 2 features: 
-A Sort Command 
-List all persons automatically after each add, delete and sort operation
Add previously removed PersonNotFoundException

### CODE REVIEW

File: src/seedu/addressbook/data/AddressBook.java [hunk 1]
```diff
@@ -128,4 +129,9 @@ public boolean equals(Object other) {
                         && this.allPersons.equals(((AddressBook) other).allPersons)
                         && this.allTags.equals(((AddressBook) other).allTags));
     }
-}
+
+    /**
+     * Sorts the address book based on persons' names in alphabetical order (based on ASCII value).
+     */
+    public void sort() { allPersons.sort(); }
```
Comment (okkhoy) [inline src/seedu/addressbook/data/AddressBook.java:136]: Please check the coding standards, how you write methods with a single line of code.

File: src/seedu/addressbook/data/person/PersonComparatorByName.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
+import java.util.Comparator;
+
+/**
+ * A comparator class that compares 2 persons based on their names' ASCII values.
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/PersonComparatorByName.java:6]: You could always make the comparator not-specific to addressbook

**Comment by okkhoy** [comment]
@karenfrilya97 

Please check why your tests are failing.

Some comments added. Please ack & close the PR after reading comments.

**Comment by karen-celine** [comment]
Acknowledged. Thank you!

**Pull request closed without merging**